packages/test/service-load-test/src/nodeStressTest.ts

```diff
@@ -262,15 +240,18 @@ async function orchestratorProcess(
             );
 
         console.log("You may need to allow consent for this app. Re-run the tool after allowing consent.");
-        console.log(`Go here allow the app: ${loginPageUrl}`);
+        console.log(`Go here allow the app: ${loginPageUrl}\n`);
 
         throw ex;
     }
 
+    // Automatically determine driveId based on the server and user
+    const driveId = await getDriveId(loginInfo.server, "", undefined, { accessToken: odspTokens.accessToken });
+
     // Create a new file if a url wasn't provided
     const url = args.url ?? await initialize(driveId, loginInfo);
-    const estRunningTimeMin = Math.floor(2 * profile.totalSendCount / (profile.opRatePerMin * profile.numClients));
 
+    const estRunningTimeMin = Math.floor(2 * profile.totalSendCount / (profile.opRatePerMin * profile.numClients));
```

Comment:
I'm not sure I follow this math. Why the times 2 here?

Reply:
Ha yeah this was harder than I expected to figure out.  The 2x is because each client goes through alternating read/write cycles of equal duration.  So only half the time are they actually sending ops, so it takes twice as long as if they were writing the whole time